Update io.py. The old one does not work any more.

jonduan opened the pull request:
change default url for pwt table to http://www.rug.nl/ggdc/docs/

depreciation rates also change to http://www.rug.nl/ggdc/docs/

jonduan committed:
Update io.py

change default url for pwt table to http://www.rug.nl/ggdc/docs/

depreciation rates also change to http://www.rug.nl/ggdc/docs/

davidrpugh commented:
@jonduan Looks like the version number also needs to be bumped from '80' to  '90'...

jonduan committed:
Update io.py

download pwt90.dta directly from http://www.rug.nl/ggdc/docs/ using request and open write method

change pwt version from  8.0 to 9.0

jonduan commented:
Yes, I change the version number from 8.0 to 9.0 and download pwt90.dta directly instead of pwt90.zip.
